packages/intrepid2/src/Orientation/Intrepid2_OrientationToolsDefMatrixData.hpp

```diff
@@ -91,8 +91,8 @@ namespace Intrepid2 {
         if(dynamic_cast<const typename NodalBasisFamily<ExecutionSpace, OutputValueType, PointValueType>::HGRAD_QUAD*>(basis)) {
```

Comment:
Totally agree, I hate that piece of code as well.
However the issue there is not to get the same basis of a different order, but to to get the same basis defined on host, instead of on device.

Reply:
Oh, yes, I see that now: the distinction is in the execution space, not the polynomial order.